packages/strapi-plugin-content-manager/admin/src/translations/pl.json

```diff
@@ -44,21 +67,32 @@
   "containers.List.pluginHeaderDescription": "{label} elementów znalezionych",
   "containers.List.pluginHeaderDescription.singular": "{label} element znaleziony",
   "containers.ListPage.displayedFields": "Wyświetlone atrybuty",
+  "containers.ListSettingsView.modal-form.edit-label": "Edytuj etykietę",
+  "containers.SettingPage.add.field": "Wstaw inne pole",
+  "containers.SettingPage.add.relational-field": "Wstaw inne pole relacyjne",
   "containers.SettingPage.attributes.description": "Zdefiniuj kolejność atrybutów",
   "containers.SettingPage.listSettings.title": "Lista (ustawienia)",
+  "containers.SettingPage.editSettings.entry.title": "Tytuł wpisu",
+  "containers.SettingPage.editSettings.entry.title.description": "Ustaw wyświetlane pole swojego wpisu",
   "containers.SettingsPage.Block.generalSettings.title":	"Ogólne",
+  "containers.SettingPage.layout": "Układ",
   "containers.SettingPage.addField": "Dodaj nowy atrybut",
   "containers.SettingPage.editSettings.description": "Przeciągnij i upuś pola by zbudować układ",
   "containers.SettingPage.editSettings.title": "Edycja (ustawienia)",
   "containers.SettingPage.relations": "Pola relacyjne",
   "containers.SettingPage.addRelationalField": "Dodaj nowe pola relacyjne",
+  "containers.SettingPage.settings": "Ustawienia",
+  "containers.SettingPage.view": "Widok",
   "containers.SettingPage.attributes": "Pola atrybutów",
-  "containers.SettingPage.listSettings.description": "Skonfiguruj opcje dla tego typu zawartości",
-  "containers.SettingPage.pluginHeaderDescription": "Skonfiguruj specificzne ustawienia tego Typu Zawartości",
+  "containers.SettingPage.listSettings.description": "Skonfiguruj opcje dla tego modelu",
```

Comment:
I think here it is better to keep the old translation `content type` as `model` is not in our wording. (They will soon be changed to either `collectionType` or `singleType`.

I'll let @Aurelsicoko decide if whether or not `model` is ok.

Reply:
sure thing. Thanks for clarification. In PL translations we had content types called translated as content types in pl and in other places as models... So I made this decision. Model sems ok, but I'll wait for @Aurelsicoko